sql/internet_outages/italy_covid19_outage_v1/query.sql

```diff
@@ -0,0 +1,193 @@
+DECLARE country_code STRING DEFAULT "IT";
+DECLARE fromdate DATE DEFAULT '2020-03-01'; -- '2019-12-01'
+DECLARE todate DATE DEFAULT '2020-03-02'; --'2020-03-31';
+
+-- The minimum size of a bucket for it to be published.
+DECLARE MIN_NUMBER_CLIENTS INT64 DEFAULT 5000;
+
+-- Note: udf.udf_json_extract_int_map map doesn't work in this case as it expects an INT -> INT
+-- map, while we have a STRING->int map
+CREATE TEMP FUNCTION
+  udf_json_extract_string_to_int_map (input STRING) AS (ARRAY(
+    SELECT
+      STRUCT(CAST(SPLIT(entry, ':')[OFFSET(0)] AS STRING) AS key,
+             CAST(SPLIT(entry, ':')[OFFSET(1)] AS INT64) AS value)
+    FROM
+      UNNEST(SPLIT(REPLACE(TRIM(input, '{}'), '"', ''), ',')) AS entry
+    WHERE
+      LENGTH(entry) > 0 ));
+     
+
+-- Get a stable source for DAUs.
+WITH DAUs AS (
+    SELECT
+        submission_date as date,
+        COUNT(*) as client_count
+    FROM
+        telemetry.clients_daily
+    WHERE
+        submission_date >= fromdate
+        AND submission_date <= todate
+        AND country = country_code
+    GROUP BY 1
```

Comment:
For this specific dataset, doesn't change anything because we're always over the `MIN_NUMBER_CLIENTS`, but you're right, it's better to add it to be formally correct.